playbooks/pki.yml

```diff
@@ -212,16 +212,16 @@
             ownca_privatekey_passphrase: "{{ secret_ca_passphrase }}"
             ownca_not_after: "{{ cert_days_valid_for | default(omit) }}"
             ownca_not_before: "+0s"
-            path: "{{ hostvars.localhost.pki_dir }}/certificates/{{ inventory_hostname }}.crt"
+            path: "{{ hostvars.localhost.pki_dir }}/certificates/{{ inventory_hostname | ansible.builtin.hash }}.crt"
           delegate_to: localhost
 
         - name: Generate PKCS12 keystores
           community.crypto.openssl_pkcs12:
             action: export
-            path: "{{ hostvars.localhost.pki_dir }}/certificates/{{ inventory_hostname }}.p12"
+            path: "{{ hostvars.localhost.pki_dir }}/certificates/{{ inventory_hostname | ansible.builtin.hash }}.p12"
             passphrase: "{{ p12_passphrase }}"
-            friendly_name: "{{ inventory_hostname }}"
+            friendly_name: "{{ inventory_hostname | ansible.builtin.hash }}"
```

Comment:
That's not very friendly anymore is it? :)
IS that one also causing you problems? It should not.